app/views/layouts/mailer.text.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
there's a few mailer files, were these auto generated or did you have a purpose for them?